src/D2L.CodeStyle.Analyzers/Extensions/Microsoft.CodeAnalysis.cs

```diff
@@ -9,40 +9,6 @@
 
 namespace D2L.CodeStyle.Analyzers.Extensions {
 	internal static partial class RoslynExtensions {
-		/// <summary>
-		/// Get the declaration syntax for a symbol. This is intended to be
-		/// used for fields and properties which can't have multiple
-		/// declaration nodes.
-		/// </summary>
-		public static T GetDeclarationSyntax<T>( this ISymbol symbol )

```

Comment:
Don't really like this helper because

* It hides that it is doing the equivalent `.Single()` (would rather see that explicitly)
* The exception from `.Single()` is fine
* It's sort of hiding the cast